react-app/src/components/complement/Menu.js

```diff
@@ -0,0 +1,69 @@
+import React, { Component } from 'react';
+import {
+  Dropdown,
+  DropdownToggle,
+  DropdownMenu,
+  DropdownItem
+} from 'reactstrap';
+import MenuIcon from '@material-ui/icons/Menu';
+
+const style = {
+  icon: {
+    float: 'right'
+  },
+  logo: {
+    width: '20px',
+    height: '30px',
+    margin: '0 5px'
+  }
+};
+
+export default class Menu extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      dropdownOpen: false
+    };
+    this.toggle = this.toggle.bind(this);
```

Comment:
should not use any more `this.function = this.function.bind(this)` in the latest versions of React